compiler/src/iree/compiler/Preprocessing/Common/PadToIntrinsics.cpp

```diff
@@ -30,26 +31,116 @@ namespace mlir::iree_compiler::Preprocessing {
 namespace {
 
 static Value getPaddedValue(RewriterBase &rewriter, Location loc,
-                            Value padSource, ArrayRef<int64_t> padding) {
-  auto sourceType = cast<RankedTensorType>(padSource.getType());
+                            Value padSource, ArrayRef<OpFoldResult> padding) {
+  auto sourceType = padSource.getType().cast<RankedTensorType>();
```

Comment:
This way is being deprecated. What is there before is the way the casting needs to be done.

Reply:
done!